airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/conftest.py

```diff
@@ -162,32 +162,12 @@ def previous_connector_image_name_fixture(image_tag, inputs) -> str:
 
 
 @pytest.fixture(name="previous_connector_docker_runner")
-def previous_connector_docker_runner_fixture(previous_connector_image_name, tmp_path) -> ConnectorRunner:
+def previous_connector_docker_runner_fixture(previous_connector_image_name, base_path) -> ConnectorRunner:
     """Fixture to create a connector runner with the previous connector docker image.
     Returns None if the latest image was not found, to skip downstream tests if the current connector is not yet published to the docker registry.
     Raise not found error if the previous connector image is not latest and expected to be published.
     """
-    try:
-        return ConnectorRunner(previous_connector_image_name, volume=tmp_path / "previous_connector")
-    except (errors.NotFound, errors.ImageNotFound) as e:
-        if previous_connector_image_name.endswith("latest"):
-            logging.warning(
-                f"\n We did not find the {previous_connector_image_name} image for this connector. This probably means this version has not yet been published to an accessible docker registry like DockerHub."
-            )
-            return None
-        else:
-            raise e
-
-
-@pytest.fixture(scope="session", autouse=True)
-def pull_docker_image(acceptance_test_config) -> None:
-    """Startup fixture to pull docker image"""
-    image_name = acceptance_test_config.connector_image
-    config_filename = "acceptance-test-config.yml"
-    try:
-        ConnectorRunner(image_name=image_name, volume=Path("."))
-    except errors.ImageNotFound:
-        pytest.exit(f"Docker image `{image_name}` not found, please check your {config_filename} file", returncode=1)
```

Comment:
This is not needed as Dagger will use the container passed by airbyte-ci, build the container according to the local docker file, or pull the image if it's not `:dev`

Reply:
Love to see try and ifs get deleted